developer_manual/design/navigation.rst

```diff
@@ -387,3 +438,43 @@ Various information
 
 * You can add the ``icon-loading-small`` class to any ``li`` element to set it in a `loading` state.
 * Every element as a ``min-height`` of 44px as that is the minimum recommended touch target. It also helps with clickability and separation on desktop environments.
+
+..  _settings:
+
+=========
+Settings
+=========
+
+Introduction
+-------------
+
+To create a settings area create a div with the id ``app-settings`` inside the ``app-navigation`` div.
+
+* The data attribute ``data-apps-slide-toggle`` slides up a target area using a jQuery selector and hides the area if the user clicks outside of it.
```

Comment:
Is the toggling done by server code or do the app developer need to implement it themselves?

Reply:
server :)